datadog.gemspec

```diff
@@ -12,7 +12,8 @@ Gem::Specification.new do |spec|
   spec.name                  = 'datadog'
   spec.version               = Datadog::VERSION::STRING
   # required_ruby_version should be in a single line due to test-head workflow `sed` to unlock the version
-  spec.required_ruby_version = [">= #{Datadog::VERSION::MINIMUM_RUBY_VERSION}", "< #{Datadog::VERSION::MAXIMUM_RUBY_VERSION}"]
+  spec.required_ruby_version = [">= #{Datadog::VERSION::MINIMUM_RUBY_VERSION}",
+                                "< #{Datadog::VERSION::MAXIMUM_RUBY_VERSION}"]
```

Comment:
I believe this breaks a `sed` script in one of our workflows, meaning we should keep this in one line.

Did Robocop change this automatically for you?

Reply:
It does indeed look like an auto-fix. I'll restore this and add a rubocop exception, to avoid it happening accidentally again.